engine/src/test/resources/testConfigPackage/testConfigPackageSrc/META-INF/properties.xml

```diff
@@ -48,5 +48,9 @@
     <name>dt.test.2</name>
     <value>config-package</value>
   </property>
+  <property>
+    <name>dt.test.7</name>
```

Comment:
What is the purpose of this property? It seems to serve the same purpose as dt.test.2?